collections/_infographics/policies/fit-for-55.md

```diff
@@ -1,13 +1,134 @@
+* [Revision of the Energy Taxation Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0563) (*Energy Taxation Directive*, ETD)
+
+### Objectives and regulation
+
+#### Commitments to reduce emissions (in sectors not burdened by market mechanisms)
+
+* [Revision of the Effort Sharing Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0555)
+
+#### Energy
+
+* [Revision of the Renewable Energy Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0557) (*Renewable Energy Directive*, RED)
+* [Revision on reducing methane emissions in the energy sector and amending regulation (EU) 2019/942](https://eur-lex.europa.eu/legal-content/cs/TXT/?uri=CELEX%3A52021PC0805)
+* [Revision of the regulation on the internal markets for renewable and natural gas and hydrogen](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=COM%3A2021%3A804%3AFIN)
+* [Revision of the directive on common rules for the internal markets in renewable and natural gas and in hydrogen](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=COM%3A2021%3A803%3AFIN)
+
+#### Energy efficiency
+
+* [Revision of the Energy Efficiency Directive](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0558) (*Energy Efficiency Directive*, EED)
+* [Revision of the Energy Performance of Buildings Directive](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=COM%3A2021%3A802%3AFIN) (*Energy Performance of Buildings Directive*, EPBD)
+
+#### Transportation
+
+<!--* [Stricter emission regulations for cars and vans](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0556)-->
+* [New infrastructure for alternative fuels](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0559)
+* [Initiative for more sustainable aviation fuels ReFuelEU](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0561)
+* [FuelEU Cleaner Marine Fuels Initiative](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0562)
+
+#### Landscape and ecosystems
+
+* [EU Forest Strategy](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021DC0572)
+* [Revision of the Land Use and Forestry Regulation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0554)
+
+### Support measures
+
+* Establishment of the [Social Climate Fund](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A52021PC0568)
+* Strengthening [Modernization](https://ec.europa.eu/clima/eu-action/funding-climate-action/modernisation-fund_en) and [Innovation Fund](https://ec.europa.eu/clima/eu-action/funding-climate-action/innovation-fund_en)
+<!--
+For more information on the proposed measures, go to our infographic [Fit for 55: an overview of the proposed measures](/infografiky/fit-for-55-opatreni).
+-->
+
+## Resources and notes
+
+### Who to contact next
+
+Here are some of the experts who deal with the 'Fit for 55' package:
```

Comment:
I would write "work with" instead of "deal with" but it is a subjective choice.

Reply:
Updated it to "work with" as suggested.